### DISCUSSION TITLE
Compile midi_adapter_flex

### DISCUSSION THREAD

**Discussion by ajabadia** [probonopd/MiniDexed#612] [Q&A] [answered]
I am trying to compile midi_adapter_flex https://github.com/rsta2/pico/tree/main/midi_adapter_flex but I am unable (I am not a programmer and following the tutorials does not work).
In the wiki (minidexed) there is a link to the compiled version of midi_adapter (without "flex") for the raspberry pico, which works perfectly.
I would like to propose to put the compiled version of the "flex" version.
The difference is that the "flex" uses two UART ports instead of one, which allows (or should allow) to add classic midi ports (DIN 5p) to the pico, leaving free pins of the raspberry pi for other options.

**Comment by diyelectromusic** [comment, marked as answer]
This isn't part of MiniDexed, so you might need to raise this in Rene's (rsta2) GitHub repository...

Kevin

**Comment by ajabadia** [comment]
Yes, you are right, I was just saying that because the compiled version of the simple version is on this page (and it is not on the original author's page or I don't see it) and I thought it might be a good idea to provide both versions.

**Reply by diyelectromusic** [reply]
Oh I see - I hadn't realised we hosted a build of it :)
I guess we could!  Maybe someone could take a look then... @probonopd did you build the other one?

Kevin

**Reply by probonopd** [reply]
So far the compiled simple version is all I have run. If I remember correctly, I had not built it myself.
